data-mermaid/mermaid-webapp
# Add glossary/helper text to inputs

trello card [here](https://trello.com/c/n1efESGS/350-glossary-help-for-definitions-tide-current-etc)

summary

Use the `HelperText` for the input components to show additional info about that input (whenever helper text was available).

Helper text was added to :
1. all collect record inputs
2. site
3. management regime

To test:
1. go to a page where helper text is available
2. click on info icon
3. ensure helper text appears and hides when info icon is clicked
